src/renderer/src/lib/locale/en/errors.json

```diff
@@ -1,4 +1,5 @@
 {
+	"appNameExistsError": "An app with the same name is already installed",
```

Comment:
Maybe add "please choose a different name." or similar to be clear that the app can still be installed but the user just needs to pick a different name. For example

"An app with the same name is already installed. Please choose a different name."

Also I think we should at some point check whether a name is already used beforehand, i.e. disable the install button if the chosen name is already in use. But no need to do this right now.

Reply:
100% agree, I pushed a fix with this message